src/CIG-Tests/CigPharoFunctionTemplateTest.class.st

```diff
@@ -102,7 +102,7 @@ CigPharoFunctionTemplateTest >> testGenerateMethod [
 		assert: template generate
 		equals: 'getBounds
 
-	self ffiCall: #(xx_TRect* xx_TView_getBounds(xx_TView* self))'
+	^ self ffiCall: #(xx_TRect* xx_TView_getBounds(xx_TView* self))'
```

Comment:
This case is good, too. The C function returns a "xx_TRect*" and the Pharo code has to return it with ^